libs/designsystem/page/src/page.component.ts

```diff
@@ -380,7 +389,9 @@ export class PageComponent
     @Optional()
     private navCtrl: NavController,
     private ionicElementPartHelper: IonicElementPartHelper,
-    public translations: TranslationService
+    public translations: TranslationService,
+    private htmlDocTitle: Title,
+    @Optional() @Inject(KIRBY_CONFIG) private config: KirbyConfig
```

Comment:
Consider using the inject function and forego the old decorators 😎

Reply:
Again - good idea, but we are looking into migrating all of `/libs/designsystem` components at once to not have different coding styles throughout, so will postpone until then!